sources/packages/app/src/App.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { Suspense } from 'react';
+import { ExperienceBuilder, RenderField } from '@craftercms/experience-builder/react';
+import { ContentInstance } from '@craftercms/models';
+import { getModel } from './lib/api';
+import { BASE_URL } from './constants';
+import { isAuthoring } from './utils';
+import './App.css';
+
+function App() {
+  const [model, setModel] = React.useState<ContentInstance>();
+
+  React.useEffect(() => {
+    getModel().subscribe((model) => {
+      setModel(model instanceof Array ? model[0] : model);
+    });
+  }, []);
+
+  return (
+    <Suspense
+      fallback={
+        <div />
+      }
+    >
+      <div className="App" role="main">
+        {model && (
+          <ExperienceBuilder
+            isAuthoring={isAuthoring()}
+            path={model.craftercms?.path}
+          >
+            <header className="App-header">
+              <RenderField
+                model={model}
+                fieldId="headText_s"
+                component="p"
+              />
+              <RenderField
+                model={model}
+                fieldId="logo_s"
+                render={(logo, fieldId) => (<img src={`${BASE_URL}${logo}`} className="App-logo" alt="logo" />)}
+              />
```

Comment:
```suggestion
              <RenderField
                model={model}
                fieldId="logo_s"
                renderTarget="src"
                component="img"
                className="App-logo"
                alt="logo"
                render={(logo) => `${BASE_URL}${logo}`}
              />
```

Reply:
@rart I have tried the following, but the XB is not enabled while I hover the cursor (other components are working fine, the current code is also working fine). Let me know if there is something wrong with the code.
```
<RenderField
                model={model}
                fieldId="logo_s"
                renderTarget="src"
                component="img"
                componentProps={{
                  className: "App-logo",
                  alt: "logo"
                }}
                render={(logo) => `${BASE_URL}${logo}`}
              />
```

Reply:
Hi @phuongnq , the issue was caused by a style applied to the image (with class `App-logo`). There was a `pointer-events: none` style that caused the hover and click (and others) not to work properly. You can take a look at https://github.com/craftercms/react-blueprint/pull/2/files#diff-7f8d10e8f03b240d4556ab4e56fcdf069d1be4b7f0d13ae7c5d6326d54d4bf1cL7